bin/fin

```diff
@@ -672,6 +672,25 @@ _remove_containers ()
 	fi
 }
 
+# Remove containers for removed projects.
+_remove_containers_removed_project_files ()
```

Comment:
This function name is not good and difficult to tell what it is about.
Let's change it to `_remove_dangling_containers`